test/test_files/copy/copy_gcs.test

```diff
@@ -0,0 +1,30 @@
+-GROUP CopyGcsTest
+-DATASET CSV tinysnb
+
+--
+
+# The purpose of this test is to make sure that we correctly initialize the rel table for the
+# newly added nodes (E.g. we need to set the column entry to NULL and create listHeader for
+# each newly added node).
+-CASE CopyGcsAnonymousTest
+
+-STATEMENT create node table person 
+             (ID INt64, fName StRING, gender INT64, isStudent BoOLEAN, 
+              isWorker BOOLEAN, age INT64, eyeSight DOUBLE, birthdate DATE,
+              registerTime TIMESTAMP, lastJobDuration interval, workedHours INT64[], 
+              usedNames STRING[], courseScoresPerTerm INT64[][], grades INT64[4], height float, 
+              PRIMARY KEY (ID));
+---- ok
+
+-STATEMENT create rel table knows (FROM person TO person, date DATE, meetTime
+                                                          TIMESTAMP, validInterval INTERVAL,
+                                                          comments STRING[], MANY_MANY)
+---- ok
+-STATEMENT create rel table meets (FROM person TO person, MANY_ONE);
+---- ok
+-STATEMENT COPY person FROM "gs://anonymous@kuzu-test/tinysnb/vPerson.csv"(HEADER=true);
+---- ok
+-STATEMENT COPY knows FROM "gs://anonymous@kuzu-test/tinysnb/eKnows.csv"(HEADER=true);
```

Comment:
Remove "(HEADER=true)" in copy knows statement as the file doesn't contain a header.